src/mam4xx/vertical_interpolation.hpp

```diff
@@ -14,8 +14,9 @@ using View1DInt = DeviceType::view_1d<int>;
 
 KOKKOS_INLINE_FUNCTION
 void vert_interp(const ThreadTeam &team, int levsiz, int pver,
-                 const View1D &pin, const ConstView1D &pmid,
-                 const View1D &datain, const View1D &dataout) {
+                 const ConstView1D &pin, const ConstView1D &pmid,
+                 const View1D &datain, const View1D &dataout,
+                 const int unit_factor_pin=1) {
```

Comment:
I like constViews wherever we can use them as they make the code easier to read. Can `datain` also be a `ConstView1D`? `levsiz` and `pver` can be `const`.